.github/workflows/codeql.yml

```
@@ -0,0 +1,45 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "main", "CU-*", "SRU*" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)